### ISSUE TITLE
Language tags on metadata should be preserved on edit

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/plum#589]

**Comment by eliotjordan** [comment]
Can't to replicate this issue. I've tried all variations on saving/updating, and the language metadata persists. Could you give an example?

**Comment by escowles** [comment]
I tried creating a record with bib 4785597.  But when I look at the data in Fedora (before making any updates), I don't see language tags on the title:
```
ns008:title "تضيح المقال في الرد على من انكر الاخذ من اللحية واكتحال"^^<http://www.w3.org/2001/XMLSchema#string> ,
   "Tawḍīḥ al-maqāl fī al-radd ʻalá man ankara al-akhdh min al-liḥiyah wa-al-iktiḥāl."^^<http://www.w3.org/2001/XMLSchema#string> ;
```

So I think we may not be bringing in the language tags currently.

**Comment by eliotjordan** [comment]
I see now. Tags. Not the language attribute.

**Comment by tpendragon** [comment]
Huh, good catch @escowles. I'll poke at it

**Comment by escowles** [comment]
Right, in an ideal world, we'd have language tags with script suffixes to differentiate Arabic in Arabic script from Arabic in Latin script:

```
ns008:title "تضيح المقال في الرد على من انكر الاخذ من اللحية واكتحال"@ara-Arab^^<http://www.w3.org/2001/XMLSchema#string> ,
   "Tawḍīḥ al-maqāl fī al-radd ʻalá man ankara al-akhdh min al-liḥiyah wa-al-iktiḥāl."@ara-Latn^^<http://www.w3.org/2001/XMLSchema#string> ;
```

**Comment by tpendragon** [comment]
Alright, looks like they're lost in the JSONLD record because AF#attributes doesn't return literals. Trying out the new AT to see if that fixes the problem.

**Comment by tpendragon** [comment]
Yup, the AT branch fixes that issue - but breaks a LOT of our tests (we're assuming there's strings, and really they're literals..)

**Comment by escowles** [comment]
Resolved with #712
